nupic/research/frameworks/vernon/README.md

```diff
@@ -0,0 +1,86 @@
+Vernon
+==============
+Vernon is a flexible framework to allow researchers to explore neural network models under a variety of learning paradigms including continuous, meta, and sparse learning. It is designed to be light-weight with a focus on mixin classes. 
+
+Vernon is the creation of researchers at Numenta; note therefore that all of the caveats specified in the nupic.research README still apply. 
+
+Getting started
+==============
+To get started, the key modules to be aware of are
+- [handlers.py](./handlers.py) : contains classes to structure the desired experiment (such as supervised vs continual learning)
+- [run.py](./run_experiment/run.py) and [run_with_raytune.py](./run_experiment/run_with_raytune.py) : contain functions to run more advanced experiments
+- [common_models.py](../pytorch/models/common_models.py) in our PyTorch framework : contains some common models that you can use 
```

Comment:
Change the third person to passive voice to make it more formal, from "contains some common models that you can use " to "contain common models that can be used to..."